packages/webapp/__tests__/SquadFeedPage.tsx

```diff
@@ -0,0 +1,380 @@
+  });
+});
+
+describe('invitation modal', () => {
+  const token = defaultSquad.currentMember.referralToken;
+  const defaultInvitation = `${defaultSquad?.permalink}/${token}`;
+  const openedInvitationModal = async () => {
+    renderComponent();
+    const trigger = await screen.findByLabelText('Invite a new member');
+    trigger.click();
+    await screen.findByText('Invite more members to join');
+  };
+
+  it('should show squad name', async () => {
+    await openedInvitationModal();
+    const result = await screen.findAllByText(defaultSquad.name);
+    expect(result.length).toEqual(2);
+  });
+
+  it('should show squad handle', async () => {
+    await openedInvitationModal();
+    const result = await screen.findAllByText(`@${defaultSquad.handle}`);
+    expect(result.length).toEqual(2);
+  });
+
+  it('should show squad image', async () => {
+    await openedInvitationModal();
+    const alt = `${defaultSquad.handle}'s logo`;
+    const result = await screen.findAllByAltText(alt);
+    expect(result.length).toEqual(2);
+  });
+
+  it('should show invitation link on a textfield', async () => {
+    await openedInvitationModal();
+    const input = await screen.findByRole('textbox');
+    expect(input).toHaveValue(defaultInvitation);
+  });
+
+  it('should allow textfield icon to copy link', async () => {
+    await openedInvitationModal();
+    const copy = await screen.findByTestId('textfield-action-icon');
+    copy.click();
+    expect(copyToClipboard).toHaveBeenCalledWith(defaultInvitation);
+  });
+
+  it('should allow footer button to copy link', async () => {
+    await openedInvitationModal();
+    const copy = await screen.findByText('Copy invitation link');
+    copy.click();
+    expect(copyToClipboard).toHaveBeenCalledWith(defaultInvitation);
+  });
+
+  it('should close the modal when close button is clicked', async () => {
+    await openedInvitationModal();
+    const close = await screen.findByTitle('Close');
+    close.click();
+    const header = screen.queryByText('Invite more members to join');
+    await waitFor(() => expect(header).not.toBeInTheDocument());
+  });
+});
+
+// currently unable to test the options menu - the contextify is throwing error on test, needs investigation
+// describe('squad options', () => {
+//   it('should show when squad options is clicked', async () => {
+//     renderComponent();
+//     const options = await screen.findByLabelText('Squad options');
+//     options.click();
+//     await waitFor(async () => {
+//       await screen.findByRole('menu');
+//     });
+//   });
+
+//   it('should have a feedback button on mobile devices', async () => {
+//     renderComponent();
+//     const options = await screen.findByLabelText('Squad options');
+//     options.click();
+//     const feedback = await screen.findByLabelText('squad-options-feedback');
+//     expect(feedback).toHaveClass('flex tablet:hidden');
+//     expect(feedback).toHaveAttribute('href', link);
+//   });
+// });
```

Comment:
This will block for now.
Either we can remove it and log as separate or hold this PR.

Reply:
I agree. Alright, let's remove this for now as the investigation around context menu not showing can impact many areas. Thanks for the note.

Reply:
Logged! ✅